modules/cluster-logging-about-crd.adoc

```diff
@@ -110,8 +110,8 @@ spec:
       replicas: 1
   collection: <6>
     logs:
-      type: "fluentd"
-      fluentd:
+      type: "collector"
```

Comment:
if this type remains "fluent" we can keep the ref to resources.limits, if we use vector we should consider removing